src/test/run-make/issue-36710/Makefile

```diff
@@ -1,6 +1,10 @@
-# ignore-cross-compile $(call RUN,foo) expects to run the target executable natively
+# ignore-riscv64 $(call RUN,foo) expects to run the target executable natively
```

Comment:
Why is this still ignored on RISCV?

Reply:
This is just a revert of a commit replacing the individual ignores with `ignore-cross-compile`, I did it in a separate step to make it clear what I was changing.